docs/2.develop/contracts/security/random.md

```diff
@@ -0,0 +1,68 @@
+---
+id: random
+title: Random Numbers
+---
+
+When writing smart contracts in NEAR you have access to a `random seed` that enables you to create random numbers/strings
+within your contract.
+
+This `random seed` is **deterministic and verifiable**: it comes from the validator that produced the block signing the previous
+block-hash with their private key.
+
+The way the random seed is created implies two things:
+
+- Only the validator mining the transaction **can predict** what random number will come out. **No one else**.
+Because nobody knows the validators' private key (except the validator itself).
```

Comment:
```suggestion
- Only the validator mining the transaction **can predict** which random number will come out. **No one else** could predict it because nobody knows the validator's private key (except the validator itself).
```